adoc/Harbor-Private-Registry-Administration.adoc

```diff
@@ -0,0 +1,348 @@
+:experimental:
+
+[#administration]
+== Administration
+
+=== Setup Administration Access
+
+{spr} User Interface can be accessed from a supported web browser at the location provided as `<ingress_url>` during the installation.
+Find out about the initial admin user password in <<install-passwords>.
```

Comment:
<<install-passwords> is not leading anywhere. Is this coming from the original confluence content? Maybe we need some TODO here